src/build_test.cc

```diff
@@ -3083,6 +3083,40 @@ TEST_F(BuildTest, DyndepBuildDiscoverNewInput) {
   EXPECT_EQ("touch out", command_runner_.commands_ran_[2]);
 }
 
+TEST_F(BuildTest, DyndepBuildDiscoverNewInputWithValidation) {
+  // Verify that a dyndep file can be built and loaded to discover
+  // a new input to an edge.
```

Comment:
Did/do you intend to allow dyndep files to specify validation nodes?  I don't think they need to.

If not, please add a test case that verifies the dyndep parser rejects the syntax.

Reply:
Done.